### PR TITLE
Adding blog for 3.10.0.Final

### PR DESCRIPTION
**Pull request by ronsigal**

### CODE REVIEW

File: _posts/2019-10-30-resteasy-3100final-jakarta.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+date:       Jan 15, 2020 19:04:00
+author:     Ron Sigal
+---
+Here comes the first **RESTEasy** release of 2020!
+
+RESTEasy [3.10.0.Final](https://issues.redhat.com/secure/ReleaseNote.jspa?version=12342766&styleName=Html&projectId=12310560&Create=Create&atl_token=AQZJ-FV3A-N91S-UDEU_b8c753450b4d294f915fafd56bfd6bc6897c2219_lin),
+which will ship with Wildfly 19, is the first release in the 3.x line to support the **Jakarta EE** APIs. For more discussion on that topic, see the blog post
+["RESTEasy 4.4.0.Final: moving to Jakarta artifacts and much more"](https://resteasy.github.io/2019/10/30/resteasy-440final-moving-to-jakarta-artifacts-and-much-more/).
+
+Besides a number of bug fixes, it comes with a new configuration paramter, "resteasy.disable.providers", which, together with
```
Comment (asoldano) [inline _posts/2019-10-30-resteasy-3100final-jakarta.md]: s/paramter/parameter/

File: _posts/2019-10-30-resteasy-3100final-jakarta.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+layout:     post
+title:      "RESTEasy 3.9.0.Final: Jakarta artifacts, new configuration parameter"
```
Comment (asoldano) [inline _posts/2019-10-30-resteasy-3100final-jakarta.md]: it's 3.10.0.Final, not 3.9 ;-)

File: _posts/2019-10-30-resteasy-3100final-jakarta.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+["RESTEasy 4.4.0.Final: moving to Jakarta artifacts and much more"](https://resteasy.github.io/2019/10/30/resteasy-440final-moving-to-jakarta-artifacts-and-much-more/).
+
+Besides a number of bug fixes, it comes with a new configuration paramter, "resteasy.disable.providers", which, together with
+the parameters "resteasy.providers" and "resteasy.use.builtin.providers", gives a way of configuring the set of enabled
+RESTEasy builtin providers.
+
+The new parameter helped motivate exposing the RESTEasy configuration parameters in the
+Wildfly management model. For example, the jaxrs subsystem in standalone.xml can now look like
+
+    <subsystem xmlns="urn:jboss:domain:jaxrs:2.0">
```
Comment (asoldano) [inline _posts/2019-10-30-resteasy-3100final-jakarta.md]: I'm not an expert (yet) of this documentation mechanism, but I believe there're special characters to be used for having xml snippets in the page; I think you need to quote the whole block as e.g. here https://github.com/resteasy/resteasy.github.io/blame/master/_posts/2018-07-19-asynchronous-injection-in-resteasy.md#L16

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 1 above)
Comment (ronsigal) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: Good catch!!

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 2 above)
Comment (ronsigal) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: Omg. Yeah, I knew that.

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 3 above)
Comment (ronsigal) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: I went with
```
Code Blocks

To create code blocks, indent every line of the block by at least four spaces or one tab.

    <html>
      <head>
      </head>
    </html>
```
from https://www.markdownguide.org/basic-syntax/#code. It looks pretty good. Do you think it matters?
blog

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 1 above)
Comment (ronsigal) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: Done.

By the way, is there a convention about who resolves a conversation? Is it the person who raised the issue or the person who fixed it?

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 2 above)
Comment (ronsigal) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: Done.

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 1 above)
Comment (asoldano) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: No convention I'm aware of...

File: _posts/2019-10-30-resteasy-3100final-jakarta.md (hunk 3 above)
Comment (asoldano) [reply _posts/2019-10-30-resteasy-3100final-jakarta.md]: Yeah, it's good, so just leave it like you did. Thanks.

**Pull request merged**